docs/src/local/faultrecovery.md

```diff
@@ -0,0 +1,65 @@
+# Fault Recovery
+
+Sometimes panics can happen in components that provide crucial services to the rest of system. And while it is, of course, better to try and have `Result::Err` branches in place for every anticipated problem, with the use of 3rd party libraries and even some standard library functions not every possible panic can be prevented. Thus components can "fault" unexpectedly and the service their provide suddenly is not available.
```

Comment:
"Thus components can "fault" unexpectedly and the service their provide suddenly is not available."